build/ci.yml

```diff
@@ -13,6 +13,9 @@ variables:
   Build.Minor: 14
   Drops.Dir: $(Build.ArtifactStagingDirectory)/drops
   Drop.Native: $(System.DefaultWorkingDirectory)/xplat
+  CI: "true"
```

Comment:
not sure if a `CI` variable is needed, or just checking the `AGENT_NAME` is good enough.

Reply:
it will be gone after e2e pipeline starts using the build script, and at this particular moment of pure greenness I'd rather not tinker with the PR any more :D